main.cpp

```diff
@@ -107,15 +125,20 @@ int main()
 	std::stringstream failed_combo_names("ARG! DAMMIT! DARN! NOO!");
 	const std::vector<std::string> failed_combos = initialize_combolist(failed_combo_names);
 
+	std::vector<std::string> wordlist;
+	initialize_wordlist("words.txt", wordlist);
```

Comment:
This uses working directory; you need to use the system we discussed to find `data/` from the executable path, which is where the text file should reside.

Reply:
I'm a bit confused as to how to go about it I have an idea but not sure about it

* keep a file in the current directory which contains a path to `data/` or anything else needed and just access them that way

Reply:
Will fix, I was a bit confused how to go about it but I got it